content/en/docs/2024.3/Reference/data-types/ssh/authentication/SshCredentials.md

```diff
@@ -1,12 +1,67 @@
 ---
 title: "SshCredentials"
 linkTitle: "SshCredentials"
-description: "Used to represent details required to authenticate with a server."
+description: "Any data type used to represent details required for authentication when working with SSH."
 weight: 1
 ---
 
 # {{% param title %}}
 
 <p class="namespace">(Cortex.DataTypes.Ssh.Authentication.SshCredentials)</p>
 
-{{< workinprogress >}}
+{{% alert type="information" title="Information" %}} Improvements to this page are planned for the future. {{% /alert %}}
+
+## Summary
+
+Any data type used to represent details required for authentication when working with SSH.
+
+| | |
+|-|-|
+| **Category:**          | Ssh                                                      |
+| **Name:**              | `SshCredentials`                                         |
+| **Full Name:**         | `Cortex.DataTypes.Ssh.Authentication.SshCredentials`     |
+| **Alias:**             | N/A                                                      |
+| **Description:**       | Any data type used to represent details required for authentication when working with SSH. |
+| **Default Value:**     | null                                                     |
+| **Can be used as:**    | `SshCredentials`, `ISshCredentials`, `Object`, `dynamic` |
+| **Can be cast to:**    | N/A                                                      |
+
+## Remarks
+
+### Property Editor Support
+
+- The Expression Editor is available for [Input][] properties where the data type is `SshCredentials`.
+- The Literal Editor is not available for [Input][] properties where the data type is `SshCredentials`.
+- The Variable Editor is available for [Input][], [InputOutput][] and [Output][] properties where the data type is `SshCredentials`.
+
+### Known Limitations
+
+None
```

Comment:
Add SSH?

Reply:
Not in the scope of this PBI from Arnold